packages/@angular/cli/commands/version-check.ts

```diff
@@ -0,0 +1,37 @@
+const Command = require('../ember-cli/lib/models/command');
```

Comment:
Please remove this command. There's no added value if we have it in generate, build and serve etc.